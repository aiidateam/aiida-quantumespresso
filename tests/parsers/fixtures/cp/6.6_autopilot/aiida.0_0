           1  -17.174379109009372        0.0000000000000000        0.0000000000000000
CG1      -17.1743791090      -17.1743839820      -17.1743840284      -17.1743840283
CG2 1.0000000 0.3000000 0.2734494 0.4999882
           2  -17.174384028253794        0.0000000000000000        0.0000000000000000
CG1      -17.1743840283      -17.1743841671      -17.1743843595      -17.1743843594
CG2 1.0000000 0.5468989 0.3103656 0.4998717
           3  -17.174384359393361        0.0000000000000000        0.0000000000000000
CG1      -17.1743843594      -17.1743844164      -17.1743844324      -17.1743844327
CG2 1.0000000 0.6207311 0.4227147 0.5021966
           4  -17.174384432687461        0.0000000000000000        0.0000000000000000
CG1      -17.1743844327      -17.1743844299      -17.1743844623      -17.1743844635
CG2 1.0000000 0.8454295 0.4132247 0.5196914
           5  -17.174384463458303        0.0000000000000000        0.0000000000000000
CG1      -17.1743844635      -17.1743844561      -17.1743844711      -17.1743844713
CG2 1.0000000 0.8264495 0.3442924 0.5118901
           6  -17.174384471275090        0.0000000000000000        0.0000000000000000
CG1      -17.1743844713      -17.1743844720      -17.1743844735      -17.1743844732
CG2 1.0000000 0.6885848 0.3773207 0.4332206
