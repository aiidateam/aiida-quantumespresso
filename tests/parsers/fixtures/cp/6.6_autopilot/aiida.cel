     10  0.00224956
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
     12  0.00239470
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
     15  0.00261240
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
     18  0.00283009
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
     21  0.00304779
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
     24  0.00326549
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
     27  0.00348319
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
     30  0.00370089
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000
