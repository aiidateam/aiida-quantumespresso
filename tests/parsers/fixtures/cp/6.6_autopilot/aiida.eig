  STEP:     10       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.49  -12.73   -8.23   -6.42
  STEP:     12       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.53  -12.77   -8.26   -6.47
  STEP:     15       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.37  -12.67   -8.16   -6.39
  STEP:     18       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.32  -12.66   -8.12   -6.38
  STEP:     21       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.33  -12.70   -8.13   -6.39
  STEP:     24       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.20  -12.63   -8.08   -6.34
  STEP:     27       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.20  -12.60   -8.08   -6.36
  STEP:     30       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
  -24.15  -12.57   -8.05   -6.35
