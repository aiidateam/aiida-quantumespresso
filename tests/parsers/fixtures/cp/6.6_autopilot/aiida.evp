     10  2.249562E-03  6.361068E-04  0.000000E+00  2.909040E+02        -17.17438447        -17.17438447        -17.17023890        -17.16960279  1.000000E+03      -0.00000
     12  2.394695E-03  2.021100E-04  0.000000E+00  3.076704E+02        -17.17412415        -17.17412415        -17.16973964        -17.16953753  1.000000E+03      -0.00000
     15  2.612395E-03  7.835814E-05  0.000000E+00  3.334343E+02        -17.17442201        -17.17442201        -17.16967035        -17.16959199  1.000000E+03      -0.00000
     18  2.830095E-03  3.436010E-04  0.000000E+00  3.103230E+02        -17.17433368        -17.17433368        -17.16991137        -17.16956777  1.000000E+03      -0.00000
     21  3.047794E-03  2.497460E-04  0.000000E+00  3.116892E+02        -17.17424329        -17.17424329        -17.16980152        -17.16955177  1.000000E+03      -0.00000
     24  3.265494E-03  1.373922E-04  0.000000E+00  3.212534E+02        -17.17430130        -17.17430130        -17.16972323        -17.16958583  1.000000E+03      -0.00000
     27  3.483193E-03  2.988302E-04  0.000000E+00  2.933242E+02        -17.17404099        -17.17404099        -17.16986092        -17.16956209  1.000000E+03      -0.00000
     30  3.700893E-03  2.821960E-04  0.000000E+00  2.797270E+02        -17.17382794        -17.17382794        -17.16984165        -17.16955945  1.000000E+03      -0.00000
