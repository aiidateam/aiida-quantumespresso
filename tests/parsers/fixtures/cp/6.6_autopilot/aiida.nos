     10  2.249562E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     12  2.394695E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     15  2.612395E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     18  2.830095E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     21  3.047794E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     24  3.265494E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     27  3.483193E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     30  3.700893E-03      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
