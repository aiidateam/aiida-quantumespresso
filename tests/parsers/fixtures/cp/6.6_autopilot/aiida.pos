     10  0.00224956
     0.55092489705446E+00     0.89015870064501E+00     0.10589905087984E+01
    -0.38553733104569E+00    -0.43136591605268E+00    -0.14265117462520E+01
    -0.31806693537555E+00    -0.50542901591202E+00     0.41024598359085E+00
     12  0.00239470
     0.55272147991935E+00     0.89175857893172E+00     0.10665465827273E+01
    -0.39465687794958E+00    -0.43040754600185E+00    -0.14304085998814E+01
    -0.31760925012311E+00    -0.50558890643312E+00     0.41001728232213E+00
     15  0.00261240
     0.55613611707251E+00     0.89534221684383E+00     0.10784454410845E+01
    -0.40871477308902E+00    -0.42861229794304E+00    -0.14362163912582E+01
    -0.31694404649897E+00    -0.50592508680630E+00     0.40963659063586E+00
     18  0.00283009
     0.56014821010685E+00     0.89997817283135E+00     0.10902564612997E+01
    -0.42246533537203E+00    -0.42646027342670E+00    -0.14414756310823E+01
    -0.31633539217093E+00    -0.50634933558779E+00     0.40922710436142E+00
     21  0.00304779
     0.56466212510982E+00     0.90542866052214E+00     0.11016191723925E+01
    -0.43553966135344E+00    -0.42400192772153E+00    -0.14459797847172E+01
    -0.31580036648477E+00    -0.50684363767504E+00     0.40879844452030E+00
     24  0.00326549
     0.56962986127207E+00     0.91162427433000E+00     0.11130802418019E+01
    -0.44872716085447E+00    -0.42111528462533E+00    -0.14499465643365E+01
    -0.31528663127610E+00    -0.50741127873154E+00     0.40833005140841E+00
     27  0.00348319
     0.57472227351194E+00     0.91824237617228E+00     0.11241032654022E+01
    -0.46160310287754E+00    -0.41780277027431E+00    -0.14530007428545E+01
    -0.31480016066465E+00    -0.50803194224362E+00     0.40783199859077E+00
     30  0.00370089
     0.57989297408593E+00     0.92507217560154E+00     0.11343356863191E+01
    -0.47379308119608E+00    -0.41410543742723E+00    -0.14550518349346E+01
    -0.31436145580562E+00    -0.50868988801089E+00     0.40732066553847E+00
