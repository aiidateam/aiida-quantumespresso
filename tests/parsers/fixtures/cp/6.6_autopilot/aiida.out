
     Program CP v.6.6 starts on  3Nov2020 at 21:57:21 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     Fft bands division:     nmany     =       1
     Reading input from aiida.in
 ADD_RULE: POWER STEERING
   Reading rule:  ELECTRON_DYNAMICS  VERLET              
 ADD_RULE: POWER STEERING
   Reading rule:  ORTHOGONALIZATION  ORTHO               
 ADD_RULE: POWER STEERING
   Reading rule:  DT                 3.D0                
 ADD_RULE: POWER STEERING
   Reading rule:  IPRINT             3                   

   Job Title: WATER


   Atomic Pseudopotentials Parameters
   ----------------------------------

   Reading pseudopotential for specie #  1 from file :
   ../pseudo/H_HSCV_PBE-1.0.upf
   file type is UPF v.1

   Reading pseudopotential for specie #  2 from file :
   ../pseudo/O_HSCV_PBE-1.0.upf
     Message from routine scan_end:
     No INFO block end statement, possibly corrupted file
   file type is UPF v.1


   Main Simulation Parameters (from input)
   ---------------------------------------
   Restart Mode       =      -1   from_scratch   
   Number of MD Steps =      30
   Print out every           10 MD Steps
   Reads from unit    =      50
   Writes to unit     =      51
   MD Simulation time step            =      10.00
   Electronic fictitious mass (emass) =      50.00
   emass cut-off                      =       3.00

   Simulation Cell Parameters (from input)
   external pressure       =            0.00 [KBar]
   wmass (calculated)      =         2493.41 [AU]
   ibrav =    1
   alat  =    10.00000000
   a1    =    10.00000000    0.00000000    0.00000000
   a2    =     0.00000000   10.00000000    0.00000000
   a3    =     0.00000000    0.00000000   10.00000000

   b1    =     0.10000000    0.00000000    0.00000000
   b2    =     0.00000000    0.10000000    0.00000000
   b3    =     0.00000000    0.00000000    0.10000000
   omega =    1000.00000000

    ========================================
    |  CONJUGATE GRADIENT                  |
    ========================================
    | iterations   =           100         |
    | conv_thr     = 0.00000100000 a.u.    |
    | passop       =       0.30000 a.u.    |
    | niter_cg_restart =  20      |
    ========================================

   Energy Cut-offs
   ---------------
   Ecutwfc =   80.0 Ry,      Ecutrho =  320.0 Ry,      Ecuts =  320.0 Ry
   Gcutwfc =   14.2     ,    Gcutrho =   28.5          Gcuts =   28.5
   NOTA BENE: refg, mmx =   0.050000  7680
   Orthog. with Gram-Schmidt
   Electron dynamics with conjugate gradient
   Electron dynamics : the temperature is not controlled
   initial random displacement of el. coordinates with  amplitude=  0.020000

   Electronic states
   -----------------
   Number of Electrons=     8, of States =     4
   Occupation numbers :
   2.00 2.00 2.00 2.00


   Exchange and correlations functionals
   -------------------------------------
     Exchange-correlation=  SLA  PW   PBX  PBC
                           (   1   4   3   4   0   0   0)


   Ions Simulation Parameters
   --------------------------
   Ions are allowed to move
   Ions dynamics with newton equations
   the temperature is computed for     9 degrees of freedom
   ion dynamics with fricp =  0.0000 and greasp =  1.0000
   Ionic position (from input)
   sorted by specie, and converted to real a.u. coordinates
   Species   1 atoms =    2 mass =      1822.89 (a.u.),         1.00 (amu) rcmax =   0.50 (a.u.)
        0.571642     0.943352     0.965650
       -0.243397    -0.435015    -1.378745
   Species   2 atoms =    1 mass =     29166.22 (a.u.),        16.00 (amu) rcmax =   0.50 (a.u.)
       -0.328246    -0.508525     0.413094
   Ionic position read from input file

   All atoms are allowed to move
   Ionic temperature is not controlled


   Cell Dynamics Parameters (from STDIN)
   -------------------------------------
   Starting cell generated from CELLDM
   Constant VOLUME Molecular dynamics
   cell parameters are not allowed to move

   Verbosity: iverbosity =  0



   Simulation dimensions initialization
   ------------------------------------

   unit vectors of full simulation cell
   in real space:                         in reciprocal space (units 2pi/alat):
   1    10.0000    0.0000    0.0000              1.0000    0.0000    0.0000
   2     0.0000   10.0000    0.0000              0.0000    1.0000    0.0000
   3     0.0000    0.0000   10.0000              0.0000    0.0000    1.0000

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        2561    2561    641                96969    96969   12053


   Real Mesh
   ---------
   Global Dimensions   Local  Dimensions   Processor Grid
   .X.   .Y.   .Z.     .X.   .Y.   .Z.     .X.   .Y.   .Z.
    60    60    60      60    60    60       1     1     1
   Array leading dimensions ( nr1x, nr2x, nr3x )   =     60    60    60
   Local number of cell to store the grid ( nrxx ) =     216000
   Number of x-y planes for each processors: 
  |  60,  60  |

   Smooth Real Mesh
   ----------------
   Global Dimensions   Local  Dimensions   Processor Grid
   .X.   .Y.   .Z.     .X.   .Y.   .Z.     .X.   .Y.   .Z.
    60    60    60      60    60    60       1     1     1
   Array leading dimensions ( nr1x, nr2x, nr3x )   =     60    60    60
   Local number of cell to store the grid ( nrxx ) =     216000
   Number of x-y planes for each processors: 
  |  60,  60  |

   Reciprocal Space Mesh
   ---------------------
   Large Mesh
     Global(ngm_g)    MinLocal       MaxLocal      Average
          48485          48485          48485       48485.00
   Smooth Mesh
     Global(ngms_g)   MinLocal       MaxLocal      Average
          48485          48485          48485       48485.00
   Wave function Mesh
     Global(ngw_g)    MinLocal       MaxLocal      Average
           6027           6027           6027        6027.00


   System geometry initialization
   ------------------------------
   ibrav =    1       cell parameters read from input file

   Matrix Multiplication Performances
   ortho mmul, time for parallel driver      =   0.00000 with    1 procs

   Constraints matrixes will be distributed block like on
   ortho sub-group =    1*   1 procs



   Pseudopotentials initialization
   -------------------------------


   Common initialization

   Specie:     1

                        dion 

   Specie:     2
   1  indv=  1   ang. mom=  0

                        dion 
   0.2201

   Cell parameters from input file are used in electron mass preconditioning
   init_tpiba2=    0.39478418

   Short Legend and Physical Units in the Output
   ---------------------------------------------
   NFI    [int]          - step index
   EKINC  [HARTREE A.U.] - kinetic energy of the fictitious electronic dynamics
   TEMPH  [K]            - Temperature of the fictitious cell dynamics
   TEMP   [K]            - Ionic temperature
   ETOT   [HARTREE A.U.] - Scf total energy (Kohn-Sham hamiltonian)
   ENTHAL [HARTREE A.U.] - Enthalpy ( ETOT + P * V )
   ECONS  [HARTREE A.U.] - Enthalpy + kinetic energy of ions and cell
   ECONT  [HARTREE A.U.] - Constant of motion for the CP lagrangian



   Wave Initialization: random initial wave-functions
   Occupation number from init
   nbnd =     4
    2.00 2.00 2.00 2.00
 Randomizing ions velocities according to tempw (INPUT VELOCITIES DISCARDED) 

   formf: eself=    30.31961
   formf:     vps(g=0)=  -0.0007957     rhops(g=0)=  -0.0010000
   formf:     vps(g=0)=  -0.0007862     rhops(g=0)=  -0.0009756
   formf:     vps(g=0)=  -0.0007862     rhops(g=0)=  -0.0009756
   formf:     vps(g=0)=  -0.0007862     rhops(g=0)=  -0.0009756
   formf:     vps(g=0)=  -0.0007776     rhops(g=0)=  -0.0009518
   formf: sum_g vps(g)=  -1.7264663 sum_g rhops(g)=  -0.7188485
   formf:     vps(g=0)=  -0.0037910     rhops(g=0)=  -0.0060000
   formf:     vps(g=0)=  -0.0037674     rhops(g=0)=  -0.0058538
   formf:     vps(g=0)=  -0.0037674     rhops(g=0)=  -0.0058538
   formf:     vps(g=0)=  -0.0037674     rhops(g=0)=  -0.0058538
   formf:     vps(g=0)=  -0.0037505     rhops(g=0)=  -0.0057111
   formf: sum_g vps(g)=  -2.9647389 sum_g rhops(g)=  -4.3130909
   Delta V(G=0):   0.006278Ry,    0.170835eV
 PERFORMING CONJUGATE GRADIENT MINIMIZATION OF EL. STATES

  nfi     ekinc              temph  tempp     etot                 enthal               econs                econt              vnhh    xnhh0   vnhp    xnhp0

       nfi   tempp            E -T.S-mu.nbsp         +K_p #Iter
Step     1     385    -17.175705    -17.175705    -17.167045    36
Step     2     365    -17.175412    -17.175412    -17.170210    10
Step     3     343    -17.175096    -17.175096    -17.170208     7
Step     4     321    -17.174792    -17.174792    -17.170206     7
Step     5     303    -17.174534    -17.174534    -17.170204     7
Step     6     290    -17.174348    -17.174348    -17.170203     7
Step     7     283    -17.174249    -17.174249    -17.170203     7
Step     8     282    -17.174234    -17.174234    -17.170204     7
 ==================================================
  Setting tprint=.true. for this step (last of CG)
 ==================================================
NOTE: eigenvalues are not computed without ortho
Step     9     286    -17.174289    -17.174289    -17.170206     7

 ****************************************************
   Autopilot employ rules: 
    Rule event: iprint                      3
    Rule event: dt                      3.0000
 Rescaling ionic velocities with new dt
 Wow, setting tortho=.true. !
    Rule event: electron_dynamics  VERLET    
 ****************************************************

     10    0.000636106776602    0.0  290.90     -17.174384473228     -17.174384473228     -17.170238901582     -17.169602794805   0.0000   0.0000   0.0000   0.0000
     11    0.000484186493643    0.0  296.25     -17.174281197734     -17.174281197734     -17.170059459613     -17.169575273119   0.0000   0.0000   0.0000   0.0000
     12    0.000202110034999    0.0  307.67     -17.174124147670     -17.174124147670     -17.169739643899     -17.169537533864   0.0000   0.0000   0.0000   0.0000
     13    0.000060578160983    0.0  320.91     -17.174182961990     -17.174182961990     -17.169609787901     -17.169549209740   0.0000   0.0000   0.0000   0.0000
     14    0.000073990162078    0.0  330.74     -17.174375638088     -17.174375638088     -17.169662341707     -17.169588351545   0.0000   0.0000   0.0000   0.0000
     15    0.000078358140768    0.0  333.43     -17.174422008565     -17.174422008565     -17.169670351818     -17.169591993677   0.0000   0.0000   0.0000   0.0000
     16    0.000051269038285    0.0  328.63     -17.174292858531     -17.174292858531     -17.169609731833     -17.169558462795   0.0000   0.0000   0.0000   0.0000
     17    0.000132107236306    0.0  319.37     -17.174226621007     -17.174226621007     -17.169675448754     -17.169543341518   0.0000   0.0000   0.0000   0.0000
     18    0.000343600964198    0.0  310.32     -17.174333676438     -17.174333676438     -17.169911371759     -17.169567770795   0.0000   0.0000   0.0000   0.0000
     19    0.000497212703154    0.0  305.43     -17.174440430760     -17.174440430760     -17.170087829652     -17.169590616949   0.0000   0.0000   0.0000   0.0000
     20    0.000438245424830    0.0  306.28     -17.174380635106     -17.174380635106     -17.170015956073     -17.169577710648   0.0000   0.0000   0.0000   0.0000
     21    0.000249745967160    0.0  311.69     -17.174243291154     -17.174243291154     -17.169801516966     -17.169551770999   0.0000   0.0000   0.0000   0.0000
     22    0.000124133726614    0.0  318.39     -17.174213728625     -17.174213728625     -17.169676503855     -17.169552370128   0.0000   0.0000   0.0000   0.0000
     23    0.000117610727199    0.0  322.48     -17.174289277098     -17.174289277098     -17.169693708213     -17.169576097485   0.0000   0.0000   0.0000   0.0000
     24    0.000137392223421    0.0  321.25     -17.174301295651     -17.174301295651     -17.169723225287     -17.169585833064   0.0000   0.0000   0.0000   0.0000
     25    0.000132257800288    0.0  314.40     -17.174181019513     -17.174181019513     -17.169700596992     -17.169568339191   0.0000   0.0000   0.0000   0.0000
     26    0.000170685213148    0.0  304.01     -17.174055159458     -17.174055159458     -17.169722828189     -17.169552142976   0.0000   0.0000   0.0000   0.0000
     27    0.000298830220643    0.0  293.32     -17.174040985333     -17.174040985333     -17.169860923782     -17.169562093561   0.0000   0.0000   0.0000   0.0000
     28    0.000421827426026    0.0  285.16     -17.174067231861     -17.174067231861     -17.170003466311     -17.169581638885   0.0000   0.0000   0.0000   0.0000
     29    0.000410939421605    0.0  280.81     -17.173992394828     -17.173992394828     -17.169990646254     -17.169579706833   0.0000   0.0000   0.0000   0.0000

 * Physical Quantities at step:    30


                total energy =      -17.17382794241 Hartree a.u.
              kinetic energy =       12.56448 Hartree a.u.
        electrostatic energy =      -23.42788 Hartree a.u.
                         esr =        0.00135 Hartree a.u.
                       eself =       30.31961 Hartree a.u.
      pseudopotential energy =       -4.05754 Hartree a.u.
  n-l pseudopotential energy =        1.95687 Hartree a.u.
 exchange-correlation energy =       -4.20976 Hartree a.u.
           average potential =        0.00000 Hartree a.u.



   Eigenvalues (eV), kp =   1 , spin =  1

  -24.15  -12.57   -8.05   -6.35


   CELL_PARAMETERS
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000

   System Density [g/cm^3] :              0.2017098000


   System Volume [A.U.^3] :           1000.0000000000


   Center of mass square displacement (a.u.):   0.000000

   ATOMIC_POSITIONS
   H       0.57989297408593E+00     0.92507217560154E+00     0.11343356863191E+01
   H      -0.47379308119608E+00    -0.41410543742723E+00    -0.14550518349346E+01
   O      -0.31436145580562E+00    -0.50868988801089E+00     0.40732066553847E+00

   ATOMIC_VELOCITIES
   H       0.58210437808501E-03     0.76675084385057E-03     0.11130803928384E-02
   H      -0.13432679953506E-02     0.43502844707613E-03    -0.18399118317460E-03
   O       0.47572726079107E-04    -0.75111205682920E-04    -0.58068075603987E-04



   Partial temperatures (for each ionic specie) 
   Species  Temp (K)   Mean Square Displacement (a.u.)
        1   4.02E+02     4.4100E-02
        2   3.46E+01     2.2613E-04
     30    0.000282196031123    0.0  279.73     -17.173827942414     -17.173827942414     -17.169841649983     -17.169559453952   0.0000   0.0000   0.0000   0.0000

   writing restart file (with schema): ./out/cp_51.save/
     restart      :      0.01s CPU      0.01s WALL (       1 calls)


   Averaged Physical Quantities
                      accumulated      this run
   ekinc         :        0.00018       0.00018 (AU)
   ekin          :       12.64043      12.64043 (AU)
   epot          :      -31.76551     -31.76551 (AU)
   total energy  :      -17.17438     -17.17438 (AU)
   temperature   :      311.48243     311.48243 (K )
   enthalpy      :      -12.02196     -12.02196 (AU)
   econs         :      -17.16983     -17.16983 (AU)
   pressure      :        0.00000       0.00000 (Gpa)
   volume        :     1000.00000    1000.00000 (AU)



     Called by MAIN_LOOP:
     initialize   :      0.62s CPU      0.63s WALL (       1 calls)
     main_loop    :     27.16s CPU     29.37s WALL (      30 calls)
     cpr_total    :     27.17s CPU     29.37s WALL (       1 calls)

     Called by INIT_RUN:

     Called by CPR:
     cpr_md       :     27.17s CPU     29.37s WALL (      30 calls)
     move_electro :     27.12s CPU     29.32s WALL (      31 calls)

     Called by move_electrons:
     rhoofr       :      3.63s CPU      3.78s WALL (     215 calls)
     vofrho       :     21.87s CPU     23.88s WALL (     215 calls)
     dforce       :      1.24s CPU      1.27s WALL (     248 calls)
     calphi       :      0.01s CPU      0.01s WALL (     123 calls)
     nlfl         :      0.00s CPU      0.00s WALL (      31 calls)

     Called by ortho:
     ortho_iter   :      0.00s CPU      0.00s WALL (      21 calls)
     rsg          :      0.00s CPU      0.00s WALL (      21 calls)
     rhoset       :      0.00s CPU      0.00s WALL (      21 calls)
     sigset       :      0.00s CPU      0.00s WALL (      21 calls)
     tauset       :      0.00s CPU      0.00s WALL (      21 calls)
     ortho        :      0.01s CPU      0.01s WALL (      21 calls)
     updatc       :      0.00s CPU      0.00s WALL (      21 calls)

     Small boxes:

     Low-level routines:
     prefor       :      0.00s CPU      0.00s WALL (     164 calls)
     nlfq         :      0.01s CPU      0.01s WALL (      31 calls)
     nlsm1        :      0.06s CPU      0.06s WALL (     688 calls)
     nlsm2        :      0.01s CPU      0.01s WALL (      31 calls)
     fft          :      7.82s CPU      8.11s WALL (    1721 calls)
     ffts         :      1.81s CPU      1.85s WALL (     430 calls)
     fftw         :      2.25s CPU      2.35s WALL (     926 calls)
     betagx       :      0.15s CPU      0.15s WALL (       1 calls)
     qradx        :      0.00s CPU      0.00s WALL (       1 calls)
     gram         :      0.06s CPU      0.06s WALL (     196 calls)
     nlinit       :      0.58s CPU      0.59s WALL (       1 calls)
     init_dim     :      0.02s CPU      0.03s WALL (       1 calls)
     newnlinit    :      0.00s CPU      0.00s WALL (       1 calls)
     from_scratch :      0.01s CPU      0.01s WALL (       1 calls)
     strucf       :      0.02s CPU      0.02s WALL (      32 calls)
     calbec       :      0.06s CPU      0.06s WALL (     688 calls)


     CP           :     27.83s CPU     30.06s WALL


   This run was terminated on:  21:57:51   3Nov2020            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
